src/api/grainConfig.js

```diff
@@ -50,9 +62,9 @@ export function toDistributionPolicy(x: GrainConfig): DistributionPolicy {
     );
   }
 
-  const allocationPolicies = [];
+  const allocationPoliciesDeprecated: Array<AllocationPolicy> = [];
```

Comment:
Nit: I don't think there's any need for this variable--you could just push them directly into `allocationPolicies`

Reply:
Ack, I left as is, as I like thinking of these processes as a tree. One branch handles x.allocationPolicies and the other handles the deprecated policies.